packages/core-components/src/components/buttons/LanguageSelectButtonMobile.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+
+import {
+  type DropdownButtonProps,
+  Option,
+} from '@/components/buttons/DropdownButton';
+
+type LanguageSelectButtonMobileProps = DropdownButtonProps & {
+  buttonIcon: React.ReactNode;
+};
+
+const LanguageSelectButtonMobile: React.FC<LanguageSelectButtonMobileProps> = ({
+  selected,
+  options,
+  buttonIcon,
+}) => {
+  const defaultSelectedOption = options.find(
```

Comment:
looking at this prop type, I can imagine we provide a list of 2 options for this component although it only needs one at a time. Since we will be using this in our SSR NextJSApp, the component doesn't need to update its state anyway. 
IMO, the other props are redundant, the component only needs only 1 icon and a handler.